spring-webmvc/src/test/java/org/springframework/web/servlet/tags/MessageTagTests.java

```diff
@@ -389,7 +390,7 @@ void requestContext() throws ServletException {
 		assertThat(rc.getMessage("test", "default")).isEqualTo("test message");
 		assertThat(rc.getMessage("test", (Object[]) null, "default")).isEqualTo("test message");
 		assertThat(rc.getMessage("testArgs", new String[]{"arg1", "arg2"}, "default")).isEqualTo("test arg1 message arg2");
-		assertThat(rc.getMessage("testArgs", Arrays.asList(new String[]{"arg1", "arg2"}), "default")).isEqualTo("test arg1 message arg2");
```

Comment:
I think that `String[]` single value in the `List` was on purpose, so this changes the semantic of the test. Could be replaced by `Collections.singletonList(new String[]...)`

Reply:
But `new String[]` will either way be used here. Because Arrays.asList("arg1", "arg2") is the same as Arrays.asList(new String[]{"arg1", "arg2"}). The former will be converted to the later.

It will be different with the singletonList, as it will create a list of array, while the original was creating list of `arg1` and `arg2`.